frontend/src/pages/Graph/SummaryPanelNamespaceBox.tsx

```diff
@@ -610,28 +610,34 @@ export class SummaryPanelNamespaceBox extends React.Component<SummaryPanelPropTy
     }
 
     if (filters.length > 0) {
-      promiseIn = API.getNamespaceMetrics(namespace, {
-        byLabels: ['request_protocol'], // ignored by prom if it doesn't exist
-        cluster: cluster,
```

Comment:
It seems that when you set `{ ... } as <Type>`, you are forcing the object to be that type, disabling the property validations of that type. If you just remove the `as <Type>` (in this case `as IstioMetrics` in the parameter) the warning would have appeared because the cluster property does not exist. Something new to learn about Typescript :-).

Reply:
Interesting.  I would have thought the opposite, that `as <Type>` would actually ensure the property validation!